pkg/jx/cmd/update_webhooks.go

```diff
@@ -154,15 +154,14 @@ func (options *UpdateWebhooksOptions) updateRepoHook(git gits.GitProvider, repoN
 					Repo: &gits.GitRepository{
 						Name: repoName,
 					},
-					URL: webhookURL,
+					URL:         webhookURL,
+					ExistingURL: options.PreviousHookUrl,
 				}
 
 				if isProwEnabled {
 					webHookArgs.Secret = string(hmacToken.Data["hmac"])
 				}
 
-				log.Infof("Updating WebHook with new args\n")
```

Comment:
This lead to confusion when nothing actually happened (see @jtnord 's comments) and is better removed as we have sufficient logging in the `UpdateWebHook` method